exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/text/compliant/CompliantTextRecordReader.java

```diff
@@ -71,15 +79,14 @@ public CompliantTextRecordReader(FileSplit split, DrillFileSystem dfs, FragmentC
   // checks to see if we are querying all columns(star) or individual columns
   @Override
   public boolean isStarQuery() {
-    if(settings.isUseRepeatedVarChar()) {
-      return super.isStarQuery() || Iterables.tryFind(getColumns(), new Predicate<SchemaPath>() {
-        @Override
-        public boolean apply(@Nullable SchemaPath path) {
-          return path.equals(RepeatedVarCharOutput.COLUMNS);
-        }
-      }).isPresent();
-    }
-    return super.isStarQuery();
+    if (super.isStarQuery()) { return true; }
```

Comment:
not sure if I understood your suggestion. it does work with columns[] (ext csv) and named field cases (ext csvh), star and non star queries.
Since it was not function of UseRepeatedVarChar I simplified the code